storybook/src/elements/nuxeo-justified-grid/nuxeo-justified-grid.stories.js

```diff
@@ -0,0 +1,81 @@
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import { boolean, number } from '@storybook/addon-knobs';
+import { analyse } from '../../../.storybook/analysis';
+import { LIST } from '../../data/lists.data.js';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-justified-grid/nuxeo-justified-grid.js';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-justified-grid/nuxeo-justified-grid-item.js';
+
+const docs = analyse('nuxeo-justified-grid').notes;
+
+const stories = storiesOf('UI/nuxeo-justified-grid', module);
+
+stories
+  .add(
+    'Empty',
+    () => html`
+      <style>
+        * {
+          font-family: 'Open Sans', Arial, sans-serif;
+        }
+      </style>
+      <nuxeo-justified-grid .items="${LIST(0).data}"></nuxeo-justified-grid>
+    `,
+    { notes: { markdown: docs } },
+  )
+  .add(
+    'Default',
+    () => {
+      const numberOfItems = number('Number of items', 50);
+      setTimeout(() => {
+        document.querySelector('nuxeo-justified-grid')._resize();
+      }, 1000);
+      return html`
+        <style>
+          * {
+            font-family: 'Open Sans', Arial, sans-serif;
```

Comment:
We are not using this font anymore 😬

Reply:
I just copied it from somewhere else 😬  Maybe we need to review all the demos?